layout/_partial/sidebar-navigation.ejs

```diff
@@ -2,10 +2,10 @@
     <!-- User dropdown  -->
     <li class="dropdown">
         <ul id="settings-dropdown" class="dropdown-menu">
-            <% for (var i in theme.dropdown) { %>
+            <% for (var i in theme.sidebar.dropdown) { %>
                 <li>
-                    <a href="<%= theme.dropdown[i].link %>" target="_blank" title="<%= i %>">
-                        <i class="material-icons sidebar-material-icons sidebar-indent-left1pc-element"><%= theme.dropdown[i].icon %></i>
+                    <a href="<%= theme.sidebar.dropdown[i].link %>" target="_blank" title="<%= i %>">
+                        <i class="material-icons sidebar-material-icons sidebar-indent-left1pc-element"><%= theme.sidebar.dropdown[i].icon %></i>
```

Comment:
- Please surround this line with the following condition: `if(theme.sidebar.dropdown[i].icon){`